src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheckTest.java

```diff
@@ -271,17 +271,15 @@ public void testMemberModifiers() throws Exception {
         checkConfig.addAttribute("memberModifiers", "LITERAL_PRIVATE, LITERAL_PROTECTED,"
                 + " LITERAL_STATIC");
         final String[] expected = {
-            "6:13: " + getCheckMessage(MSG_KEY, "AbstractClass"),
-            "9:13: " + getCheckMessage(MSG_KEY,
-                "com.puppycrawl.tools.checkstyle.checks.coding.illegaltype."
-                    + "InputIllegalTypeMemberModifiers.AbstractClass"),
-            "16:13: " + getCheckMessage(MSG_KEY, "java.util.TreeSet"),
-            "17:13: " + getCheckMessage(MSG_KEY, "TreeSet"),
-            "23:15: " + getCheckMessage(MSG_KEY,
-                "com.puppycrawl.tools.checkstyle.checks.coding.illegaltype."
-                    + "InputIllegalTypeMemberModifiers.AbstractClass"),
-            "25:25: " + getCheckMessage(MSG_KEY, "java.util.TreeSet"),
-            "33:15: " + getCheckMessage(MSG_KEY, "AbstractClass"),
+            "5:13: " + getCheckMessage(MSG_KEY, "AbstractClass"),
+            "8:13: " + getCheckMessage(MSG_KEY,
+                "java.util.AbstractList"),
```

Comment:
Can you please make this one line since it doesn't break 100-chars limit?

Reply:
Done